### ISSUE TITLE
apply_hyper?

### ISSUE THREAD

**Issue by arianaa30** [liorsidi/ViolenceDetection_CNNLSTM#6]
I see a round of `apply_hyper` is applied only on `hocky` dataset first. What's the point? Is it for initialization of weights? I want the model to be for everything, not just hockey of course!

And another question: For each of the 30 Epochs, I see it says 1/320,...2/320. What is that 320?

**Comment by liorsidi** [comment]
@arianaa30 the goal of the hyper is to identify the architecture parameters such as dropout rate, learning loss, sequence length, etc. (not the initial weights) - check section 6.1 in the report for more info.

You can definitely run the tunning for each dataset separately and train a model, I don't advise you to union all the datasets and train one model because the datasets are very different from each other and I am afraid the model won't converge, but still worth a try.

I think that the number you see is number batches in an epoch. pay attention there is another print for the decrease of the learning rate in a plateau.
